server.js

```diff
@@ -49,7 +49,7 @@ app.get('/octos', (req, res) => {
     console.log(articles)
   })
 
-  return res.json({}).end()
+  return res.json(articles).end()
```

Comment:
@alnagaev , on the code below:
```
app.get('/octos', (req, res) => {
  feed('http://feeds.feedburner.com/Octocats', (err, articles) => {
    if (err) throw err

    // eslint-disable-next-line no-console
    console.log(articles)
  })

  return res.json(articles).end()
})
```
Articles is no available outside de feed call, so it needs to be inside that function to work properly, it will make `articles` available on callback for that function.